tutorials/pipeline/config_celltype.yaml

```diff
@@ -0,0 +1,154 @@
+# Config file for pipeline run.
```

Comment:
Can you add a 1 liner explaination about what this config is about and which tutorial is using it? Same goes for config_clinical.yaml

Reply:
Added 1 liner description about each config files.